orte/mca/errmgr/default_hnp/errmgr_default_hnp.c

```diff
@@ -749,7 +888,7 @@ static void proc_errors(int fd, short args, void *cbdata)
          * hosed - so just exit out
          */
         if (ORTE_PROC_MY_NAME->jobid == proc->jobid) {
-            ORTE_ACTIVATE_JOB_STATE(NULL, ORTE_JOB_STATE_DAEMONS_TERMINATED);
+            //ORTE_ACTIVATE_JOB_STATE(NULL, ORTE_JOB_STATE_DAEMONS_TERMINATED);
```

Comment:
Commented code?

Reply:
This is commented out when i was debugging, because when i kill a daemon, i don't want the job being terminated. I will double check.

Reply:
I checked, this is right, it prevent the hnp kill himself when it cannot reach a daemon.